pypeit/core/wavecal/autoid.py

```diff
@@ -973,144 +973,188 @@ def full_template(spec, par, ok_mask, det, binspectral, nsnippet=2, debug_xcorr=
     # Finish
     return wvcalib
 
```

Comment:
Did you check that this documentation is in the parset? I spent a long time writing it and would prefer not to lose it.

Reply:
Done.